### PR TITLE
Fix issue Number #149(Negative number is not handled correctly)

### PR DESCRIPTION
**Pull request by bhartidengada**
Fixed issue Number #149(Negative number is not handled correctly)

### CODE REVIEW

**Comment by bhartidengada** [comment]
Hello @BenjaminVanRyseghem 
I can see some build is created but not done, its there since past three days and there is no specific logs where its stuck.
Can you please look in to this?

Thanks & Regards,
Bharti

**Comment by BenjaminVanRyseghem** [comment]
The build is marked as "passed" on travis.

I just restarted to see

File: src/formatting.js [hunk 1]
```diff
@@ -539,6 +542,10 @@ function setCharacteristicPrecision(output, value, optionalCharacteristic, preci
         }
     }
 
+    if (value < 0) {
+        // Add back the minus sign
+        result = `-${result}`;
```
Comment (BenjaminVanRyseghem) [inline src/formatting.js:547]: I doubt this work when the negative value is surrounded by parenthesis 😕

Comment (bhartidengada) [reply src/formatting.js:547]: Can you share example for negative value surrounded by parenthesis and have some format.
For example :- For example:-
numbro(
-1.2
).format({
mantissa : 5, characteristic: 3
});

Comment (BenjaminVanRyseghem) [reply src/formatting.js:547]: sure

http://numbrojs.com/try-me.html?lang=en-US&value=-100&format=%7B%22thousandSeparated%22%3Atrue%2C%22negative%22%3A%22parenthesis%22%2C%22mantissa%22%3A4%7D

```js
{
    thousandSeparated: true,
    negative: "parenthesis",
    mantissa: 4
}
```

Comment (bhartidengada) [reply src/formatting.js:547]: Hello @BenjaminVanRyseghem 
example 1:-
numbro( 
-100
 ).format({ 
"characteristic":4,"negative":"parenthesis","mantissa":4
 });
"(0100,0000)"
example 2:-
numbro( 
-100
 ).format({ 
"thousandSeparated":true,"negative":"parenthesis","mantissa":4
 });
"(100.0000)"
Both the above examples working with my changes.

Comment (bhartidengada) [reply src/formatting.js:547]: @BenjaminVanRyseghem  Any update on this pull request ?

**Comment by BenjaminVanRyseghem** [comment]
Hi!

Could you please:
- squash your commits
- rewrite them in present tense
- add yourself to the AUTHORS file
- add a line in CHANGELOG
?

Thanks in advance,
Benjamin Van Ryseghem

**Comment by bhartidengada** [comment]
@BenjaminVanRyseghem 
I added list of file changes as you mentioned.
But it seems some test failure after build, can you please help me with that?
Logs is not clear to identify failing test.

**Comment by BenjaminVanRyseghem** [comment]
```
    formatting formatNumber computation computes the correct output

      Expected '.00' to be '0.00'.
```

It looks like a test is failing 😕

**Comment by bhartidengada** [comment]
@BenjaminVanRyseghem  which test is this?
What format this test is using?

**Comment by BenjaminVanRyseghem** [comment]
The failing test is located at [tests/src/formatting-tests.js:2452](https://github.com/BenjaminVanRyseghem/numbro/blob/develop/tests/src/formatting-tests.js#L2452)

**Comment by bhartidengada** [comment]
@BenjaminVanRyseghem 
Hello,
i fixed the failing test , refactor code in present tense.
Build also seems fine now :)

Thanks & Regards,
Bharti

**Comment by BenjaminVanRyseghem** [comment]
Nice :smile:

Could you squash your "code" commits so that you only have 2 commits:
- one for the code
- one with the AUTHORS/CHANGELOG
?

Thanks in advance :smile:

**Comment by bhartidengada** [comment]
@BenjaminVanRyseghem 
Done!

Regards,
Bharti

**Comment by bhartidengada** [comment]
@BenjaminVanRyseghem 
When this pull request will be merged with develop branch ?

Regards,
Bharti

**Comment by BenjaminVanRyseghem** [comment]
Sorry to insist, but could you please write your commit messages using present tense?

`Fixed` -> `Fix`

**Comment by bhartidengada** [comment]
@BenjaminVanRyseghem 
Done!

**Comment by BenjaminVanRyseghem** [comment]
@all-contributors please add @bhartidengada  for code

**Comment by BenjaminVanRyseghem** [comment]
Thanks!

**Comment by bhartidengada** [comment]
@BenjaminVanRyseghem  
When we can expect next release?

Regards,
Bharti

**Pull request merged**